.github/workflows/test_integration_playwright.yml

```diff
@@ -11,11 +11,12 @@ on:
       deployment:
         required: true
         type: choice
-        default: "govtool.cardanoapi.io"
+        default: "p80-z6b8d2f7a-ze34e4cb2-gtw.z937eb260.rustrocks.fr"
```

Comment:
Lets postpone this change, until the deployment script automatically updates the QA environment